megamek/src/megamek/common/pathfinder/LongestPathFinder.java

```diff
@@ -166,16 +167,16 @@ public Deque<MovePath> doRelax(Deque<MovePath> v, MovePath mpCandidate, Comparat
                      * Current implementation of doRelax() assumes that v is
                      * sorted in such way that this situation is impossible.
                      */
-                    System.err.println(new IllegalStateException(
-                            "Top Move Path uses more MPs than Move Path Candidate."));
+                    LogManager.getLogger().error("",
```

Comment:
I gotta wonder why we're bothering creating an exception here; it's not like we're throwing it - seems like logging just the string will serve the same purpose.